File: Languages/Python/python_study_5/page7/food.py
from menu_item import MenuItem

class Food(MenuItem):
    # Definisikan method __init__ 
    def __init__(self, name, price, calorie_count):
        self.name = name
        self.price = price
        self.calorie_count = calorie_count
    
    def info(self):
        return self.name + ': $' + str(self.price) + ' (' + str(self.calorie_count) + 'kkal)'
    
    def calorie_info(self):
        print('kkal: ' + str(self.calorie_count))

File: Languages/Ruby/ruby_study_5/page7/food.rb
require "./menu"

class Food < Menu
  attr_accessor :calorie
  
  # Definisikan method `info`
  def info
    return "#{self.name} $#{self.price} (#{self.calorie}kcal)"
  end
  
  def calorie_info
    return "#{self.name} mengandung #{self.calorie}kcal"
  end
end
